nui/mellotrainer.json

```diff
@@ -9683,13 +9683,13 @@
         }
       },
       {
-        "menuName": "Blackops",
+        "menuName": "Black Operative 1",
```

Comment:
Change these to `Blackops Solider` as I'm not sure people will understand it means blackops from `Black Operative` instead of the skin tone of the skin.